README.md

```diff
@@ -107,7 +107,9 @@ The following apps are supported by govuk-docker to some extent.
    - ⚠  collections
       * You will need to [populate the Content Store database](#mongodb) or run the live stack in order for it to work locally.
       * To view topic pages locally you still need to use the live stack as they rely on Elasticsearch data which we are yet to be able to import.
-   - ✅ collections-publisher
+   - ⚠ collections-publisher
+      * 'Checking for broken links' in step by step requires link-checker-api to be running in another process (see link-checker-api below).
+      * Previewing step by steps does not work as there is no draft stack.
```

Comment:
I wonder if we can identify the piece of work 'todo' here? When I read this, I was wondering if it referred to a missing dependency, or missing data, or something not working. Hopefully there's something simple we can do to get it working.

Reply:
I'm not sure how to fix it - would appreciate your experience. Clicking 'Preview' in a step by step in collections publisher takes me to `http://draft-origin.dev.gov.uk/my-step-by-step?token=[redacted]`, which shows a `503 Service Temporarily Unavailable` page.

I understand that step by steps are rendered by [collections](https://github.com/alphagov/collections), so tried running that in a separate terminal (`govuk-docker startup`) in case it has the same inter-connectivity issue as link-checker-api. When I refresh the preview URL I can see in the collections terminal window that a request has been made to `/my-step-by-step`, but I eventually get a  `Warning: Unable to communicate with application within 30 seconds.` and a `504 Gateway Time-out` page.

Reply:
Thanks for pairing IRL. Here's my attempt at summarising:

* `draft-origin.dev.gov.uk` isn't a thing in govuk-docker. Routing is separate to applications, so whilst it's not particularly dev friendly, it shouldn't be 'fixed' here.
* However, draft-origin.dev.gov.uk should be able to be swapped out for collections.dev.gov.uk, which is the rendering app.
* collections.dev.gov.uk also gave a 503, as it is not listed as a dependency of collections-publisher. Again, this isn't very dev friendly, but as collections and collections-publisher are decoupled from each other, it makes little sense to list collections as a dependency, and if we started to do things like that we'd essentially have to start up all frontend apps for all services.
* In a separate process, running `govuk-docker startup draft` on collections starts up collections.dev.gov.uk as expected. But the styling is a bit off. It transpires this is because the draft collections frontend relies on parts of static and parts of draft-static. This is a legitimate issue - have raised #189 to fix.

I'll go back and remove line 112 from this PR, and have a think about whether this information should be captured in a 'How to' or something.

Reply:
@ChrisBAshton that's all good, except all uses of 'collections.dev.gov.uk' should actually be 'draft-collections.dev.gov.uk' :-).

Reply:
Have removed this line and drafted a new "Interoperability of stacks" subsection in the "Stacks" section - please have a read.